bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -118,15 +118,16 @@ spec:
       {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.ingress.labels "context" $ ) | nindent 6 }}
       {{- end }}
     {{- end }}
-    {{- if or .Values.commonAnnotations .Values.grafana.ingress.annotations }}
```

Comment:
I think we should maintain this line but adding the new parameter, just to avoid an empty `annotations:` block just in case none of the conditions are true:
```
{{- if or .Values.grafana.ingress.certManager .Values.commonAnnotations .Values.grafana.ingress.annotations }}
```
and also restore the final `{{- end }}`